usr/share/rear/prep/NBKDC/default/40_prep_nbkdc.sh

```diff
@@ -0,0 +1,46 @@
+#
+# prepare stuff for NovaBACKUP DataCenter
+#
+
+# detect where and if DataCenter client is installed on the system running
+# REAR and include the needed bits in the recovery ISO
+if [ -e /Hiback ]; then
+
+        Log "Detected NovaBACKUP DC Installation"
+	
+	source $SHARE_DIR/lib/nbkdc-functions.sh
+	
+        # include DataCenter executables and configuration files 
+        COPY_AS_IS=( 
+                "${COPY_AS_IS[@]}" 
+                "${COPY_AS_IS_NBKDC[@]}" 
+                $SHARE_DIR 
```

Comment:
Please don't add here anything not NBKDC related, this happens already in the other ReaR scripts.